common/views/components/ArchiveTree/ArchiveTree.js

```diff
@@ -726,12 +742,52 @@ const NestedList = ({
   );
 };
 
+function createBasicTree({ work, toggles, workId }) {
+  const ancestorArray = getArchiveAncestorArray(work);
+  const partOfReversed = [...ancestorArray, parsePart(work)].reverse();
+  return [
+    partOfReversed.reduce(
+      (acc, curr, i, array) => {
+        return {
+          openStatus: true,
+          work: curr,
+          parentId: array[i + 1] && array[i + 1].id,
+          children:
+            i === 0
+              ? work.parts.map(part => ({
+                  work: parsePart(part),
+                  openStatus: false,
+                  parentId: work.partOf[0] && work.partOf[0].id,
+                }))
+              : [acc],
+        };
+      },
+      {
+        openStatus: true,
+        work: parsePart(work),
+        parentId: work.partOf[0] && work.partOf[0].id,
+        children: work.parts.map(part => ({
+          work: part,
+          children: part.children,
+          parentId: work.id,
+        })),
+      }
+    ),
+  ];
+}
```

Comment:
Looking at this - does this only give you the current work and it's ancestors? It's a little fiddly to tell what's happening here.